components/SocialMedia.vue

```diff
@@ -1,13 +1,11 @@
 <template>
 	<div class="social-media">
 		<h2>Follow us on social media</h2>
-		<ul>
-			<li v-for="s in socialMedia" :key="s.id">
-				<nuxt-link :to="s.href">
-					{{ s.title }}
-				</nuxt-link>
-			</li>
-		</ul>
+		<div class="social-media-icons">
+			<div v-for="s in socialMedias" :key="s.id">
+				<nuxt-link :to="s.href" v-html="s.svg" rel="external" />
```

Comment:
These need accessible labels. It might be good to create a `.sr-only` class like this: https://tailwindcss.com/docs/screen-readers/#app